charts/v1.0/cray-nls/Chart.yaml

```diff
@@ -24,7 +24,7 @@
 
 apiVersion: v2
 name: "cray-nls"
-version: 1.4.48
+version: 1.4.50-debug5
```

Comment:
Should we even be doing this here anymore? Is it just for your testing? I'd really like to remove the charts that exist in two repos ASAP.

Reply:
It is just for my testing.